css-project/style.css

```diff
@@ -0,0 +1,22 @@
+li{
+  color: blue;
+}
+
+ h2{
+    color: red;  }
```

Comment:
try to adjust the formatting to be consistent and to follow conventions e.g.:

```css
h2 {
  color: red; 
}
```

Read more about CSS conventions and guidelines from [here](http://udacity.github.io/frontend-nanodegree-styleguide/css.html)